### PR TITLE
Added queryBuilder methods: `whereIn`, `whereNotIn`, `orWhereIn`, `orWhereNotIn`

### PR DESCRIPTION
**Pull request by steef-paqt**
## 🚀 What did I build!

Added queryBuilder methods: `whereIn`, `whereNotIn`, `orWhereIn`, `orWhereNotIn`

---

## 🚦 Review

Please tick what is applicable:

- [ ] Should be functionally tested
- [x] Is covered by tests

### CODE REVIEW

File: src/QueryBuilder.php [hunk 1]
```diff
@@ -37,6 +37,46 @@ public function orWhere(string $key, string $operatorOrValue, string $value = nu
...
+
+        return $this;
+    }
+
+    /**
+     * Column with a value that does not match none of the values in a comma separated list of values.
+     */
+    public function whereNotIn(string $key, array $value): self
+    {
+        $this->query[$key]['NIN'] = implode(',', $value);
```
Comment (gerard-paqt) [inline src/QueryBuilder.php]:
```suggestion
        $this->query[$key]['nin'] = implode(',', $value);
```

File: src/QueryBuilder.php [hunk 2]
```diff
@@ -37,6 +37,46 @@ public function orWhere(string $key, string $operatorOrValue, string $value = nu
         return $this;
     }
 
+    /**
+     * Column with a value that matches one of the values in a comma separated list of values.
+     */
+    public function whereIn(string $key, array $value): self
+    {
+        $this->query[$key]['IN'] = implode(',', $value);
```
Comment (gerard-paqt) [inline src/QueryBuilder.php]:
```suggestion
        $this->query[$key]['in'] = implode(',', $value);
```

File: src/QueryBuilder.php [hunk 3]
```diff
@@ -37,6 +37,46 @@ public function orWhere(string $key, string $operatorOrValue, string $value = nu
...
+
+        return $this;
+    }
+
+    /**
+     * Column with a value that matches one of the values in a comma separated list of values.
+     */
+    public function orWhereIn(string $key, array $value): self
+    {
+        $this->query[$key]['or']['IN'] = implode(',', $value);
```
Comment (gerard-paqt) [inline src/QueryBuilder.php]:
```suggestion
        $this->query[$key]['or']['in'] = implode(',', $value);
```

File: src/QueryBuilder.php [hunk 4]
```diff
@@ -37,6 +37,46 @@ public function orWhere(string $key, string $operatorOrValue, string $value = nu
...
+
+        return $this;
+    }
+
+    /**
+     * Column with a value that does not match none of the values in a comma separated list of values.
+     */
+    public function orWhereNotIn(string $key, array $value): self
+    {
+        $this->query[$key]['or']['NIN'] = implode(',', $value);
```
Comment (gerard-paqt) [inline src/QueryBuilder.php]:
```suggestion
        $this->query[$key]['or']['nin'] = implode(',', $value);
```

**Review by gerard-paqt — APPROVED** [review]

**Review by gerard-paqt — COMMENTED** [review]
Ik heb een paar kleine comments achter gelaten

**Review by ajitjenapaqt — APPROVED** [review]

**Pull request merged**